### ISSUE TITLE
[BUG] - FSO and Game Bar not being disabled using the .bat scripts

### ISSUE THREAD

**Issue by LightShaft** [Atlas-OS/Atlas#1069]
### Description

Hi,

I wish to have FSO disabled by default and it's very obvious when it is enabled (less smoother gameplay experience, a lot of stutters)
Using the .bat scripts in the Atlas folder doesn't work. I have the cmd windows that says « Finished, FSO and Game Bar are now disabled. » but it's clearly not working as I still have pretty huge stutters and pressing win + G still launches game bar.

I managed to uninstall completely game bar using this command : Get-AppxPackage -AllUsers -PackageTypeFilter Bundle -Name "*Microsoft.XboxGamingOverlay*" | Remove-AppxPackage -AllUsers 

But the FSO issue is still there.

I did a clean install of AtlasOS on W11 Pro (french) 2 days ago, disabling FSO and Game bar was the first thing I did when I installed and it worked.
Yesterday I tried to do some tests on my own to see if there were differences (there was) and today I tried to disable it and it just doesn't do anything. There are still lots of stutters and game bar still launches when I press win + g (well, until I used the command I mentioned earlier to uninstall it)

Almost forgot, but Pineapple asked me to use this command : reg query "HKCU\System\GameConfigStore" /v "GameDVR_Enabled"
And the result was : GameDVR_Enabled    REG_DWORD    0x0
And Game Bar would still pop up (before I manually uninstall it), so my guess is that uninstalling is still necessary if the packages are present. I'm still looking to correct this FSO issue tho.

Thanks a lot.

### Steps to reproduce

Go to : C:\Windows\AtlasDesktop\3. General Configuration\FSO and Game Bar
Start « disable FSO » .bat file
Press Win + G and see if game bar still launches.

### Expected behavior

Basically the same behavior as 0.3.2 : FSO off (so fullscreen gameplay is smooth) and win + G doesn't start game bar.

### Actual behavior

FSO is never off until I do it manually on each game, and there's no way to disable game bar unless I use said command to manually uninstall the packages.

### Atlas Edition

Atlas for Windows 11 23H2

### Desktop information

-Ryzen 3600X
-16Gb 3733Mhz DDR4
-GTX 1080
-Samsung 970 evo plus NVME 2tb, that's where Windows/Atlas is installed, I have other drives but I don't think it's relevant to list them

### Additional content

I first went on Discord to see if other people had the same issue (I guess not), Pineapple asked me to open an issue on github

**Comment by izayoii7** [comment]
thank you
